a3p-integration/proposals/z:acceptance/package.json

```diff
@@ -77,9 +77,6 @@
     "@agoric/zone": "portal:../../agoric-sdk/packages/zone"
   },
   "ava": {
-    "require": [
-      "@endo/init/legacy.js"
```

Comment:
I should know more about why we were using endo legacy before approving this.

I'm also not sure about why use imports in every file rather than ava require. I sort of prefer it that way, but it hasn't been the norm here.

Reply:
The general reason seems to generally be JS assignment override mistake mitigation, albeit in a noisy way:

<details>

```console
$ for import in @endo/init @endo/init/legacy.js; do
  echo
  node --input-type=module -e "
    import '$import';
    import { inspect } from 'node:util';
    console.log('$import');
    let MyObject = () => 'not implemented';
    try {
      MyObject = function MyObject(props = {}) {
        return new.target ? Object.assign(this, props) : new MyObject(props);
      };
      (MyObject.prototype = Object.create(Object.prototype)).constructor = MyObject;
    } catch (err) { console.log('ERROR', err.message); }
    const instance = MyObject({ foo: 'bar' });
    console.log(util.inspect({
      data: {
        object: {instance},
        map: new Map([['instance', instance]]),
      }
    }, { depth: 10 }));
  "
done

@endo/init
ERROR Cannot assign to read only property 'constructor' of object '[object Object]'
{
  data: {
    object: { instance: { foo: 'bar' } },
    map: Map(1) { 'instance' => { foo: 'bar' } }
  }
}

@endo/init/legacy.js
Object <[Object: null prototype] {}> {
  data: Object <[Object: null prototype] {}> {
    object: Object <[Object: null prototype] {}> {
      instance: MyObject { foo: 'bar' }
    },
    map: Map <Object <[Object: null prototype] {}>>(1) [Map] {
      'instance' => MyObject { foo: 'bar' }
    }
  }
}
```

</details>

But AFAICT, it is not helpful here.

Reply:
IIRC some import of axios or protobufjs prompted it. I didn't realize it had any DX downsides. Given that it impairs the console messages I agree we should avoid it.

Reply:
> shouldn't it be `@endo/init/debug.js` for better output?

Good point; done.